### ISSUE TITLE
Win 8.1 - Right sliding panel issue

### ISSUE THREAD

**Issue by TheArkive** [riverar/uxstyle#6]
I saw on uxstyle.com that the Win 8 / 8.1 binary may be buggy, I just wanted to report a bug that I can easily replicate:

After initial install, applying custom themes happens no problem.  But after the first or second reboot (never got past a 2nd reboot before this issue popped up) the right side panel seems to appear and stay there, blocking the system tray and time/date panel.

Even more odd is that this panel is completely blank.  After that, subsequent reboots or theme changes have no effect.  The panel just stays there blocking the system tray, and time/date panel.

The panel is the exact same size as the one that appears when you click on the network / WIFI icon in the system tray.  The only way I can get rid of that panel is to uninstall UxStyle and reboot.

Other than that, UxStyle works perfectly on my copy of Win 8.1 Pro (as long as I never reboot).

EDIT:  Almost forgot ... I'm running x64 version of Win 8.1, and I'm running a few of the common addons for custom themes like OldNewExplorer, AeroGlass, StartIsBack.  I've messed with a few variations of installing/uninstalling those apps and rebooting.  Doesn't seem to have an effect so far.

Also installing / uninstalling is not an issue on my system (noticed that issue was still open).

**Comment by riverar** [comment]
Does using another theme resolve the issue?

**Comment by TheArkive** [comment]
:-/

Sorry to report, but the reliable bug has disappeared!  I guess that's good, but I'm having one heck of a time replicating it now.

I think at the time it was happening I had UltraUxThemePatcher installed too.

I've recently managed to make the same glitch happen on Win 8.1 but through a completely different means.  When using Windows Style Builder Beta for Win 8.1, any time I try and edit the text colors on the Time/Date control panel, and then apply that msstyle file... it does the same thing oddly enough.  The only fix is to switch to a default theme and reboot.

I know that doesn't entirely help you out... sorry for that.  Quick question:

Is UxStyle meant to be a total replacement for UltraUxThemePatcher?

In the last few weeks, I came across the command line "sfc /scannow" in order to get an easy "do-over" and not need to do a complete OS re-install.  I'm guessing now that the main theme DLL's have been restored UxStyle is working fine again.
